mp/src/game/client/hud_crosshair.h

```diff
@@ -11,6 +11,7 @@
 #pragma once
 #endif
```

Comment:
You can also get rid of the ifdef header guard, and only use `#pragma once` up top.